privacyidea/api/user.py

```diff
@@ -138,6 +144,7 @@ def delete_user(resolvername=None, username=None):
 @user_blueprint.route('/', methods=['POST'])
 @admin_required
 @prepolicy(check_base_action, request, ACTION.ADDUSER)
+@event("user_create", request, g)
```

Comment:
I'm wondering if we could somehow test the functionality of the ``user_?`` events? I think it would be interesting to see if e.g. the ``TokenEventHandler`` would allow to enroll a token for the newly created user.

Reply:
see below.